### PR TITLE
add patient portal link to admin page

### PR DESCRIPTION
**Pull request by gutiersa**
Improvements to admin page.

Add a link to the patient portal for easy access.
(This PR may be ignored, I am following Brady's video tutorial)

### CODE REVIEW

**Comment by bradymiller** [comment]
2DV

**Comment by gutiersa** [comment]
Finally, right?
@sjpadgett  you better believe it!

File: admin.php [hunk 1]
```diff
@@ -164,6 +165,7 @@ function sqlQuery($statement, $link)
                                 }
                                 if (($v_database == $database_version) && ($v_acl <= $database_acl) && ($v_realpatch == $database_patch)) {
                                     echo "  <td><a href='interface/login/login.php?site=$sfname' class='text-decoration-none'><i class='fa fa-sign-in-alt fa-lg' aria-hidden='true' data-toggle='tooltip' data-placement='top' title ='Login to site $sfname'></i></a></td>\n";
+                                    echo "  <td><a href='portal/index.php?site=$sfname' class='text-decoration-none'><i class='fa fa-sign-in-alt fa-lg' aria-hidden='true' data-toggle='tooltip' data-placement='top' title ='Login to site $sfname'></i></a></td>\n";
                                 } else {
                                     echo "  <td><i class='fa fa-ban fa-lg text-secondary' aria-hidden='true'></i></td>\n";
```
Comment (bradymiller) [inline admin.php:170]: just need to repeat above line and then this should be good to bring into codebase:
```
echo "  <td><i class='fa fa-ban fa-lg text-secondary' aria-hidden='true'></i></td>\n";
```

Comment (gutiersa) [reply admin.php:170]: Ooooh, I see, thanks. Is that why "Test/php8.2 - Nginx - MariaDB 10.6" fails??

**Comment by bradymiller** [comment]
@gutiersa , code looks great and bringing in to codebase. thanks for the contribution!

dance-bear

**Comment by gutiersa** [comment]
Yay!

**Comment by stephenwaite** [comment]
southpark_cheering

**Pull request merged**